config/betterquesting/DefaultQuests.json

```diff
@@ -261775,8 +261792,12 @@
           "isMain:1": 0,
           "simultaneous:1": 0,
           "icon:10": {
-            "id:8": "Avaritia:Orb_Armok",
+            "id:8": "betterquesting:placeholder",
```

Comment:
Why do you do this to me BQ3?

Dream, please don't merge until I fix this.

Reply:
ok